### ISSUE TITLE
An option to make meterpreter session more verbose

### ISSUE THREAD

**Issue by g0tmi1k** [rapid7/metasploit-payloads#34]
**tl;dr**: Have the option to make the foreground meterpreter session a little more verbose

- - - 

So after watching @OJ doing a talk (https://vimeo.com/108076345#t=27m22s), he says:

    "Ever got to this point here and run a command, and it said command not found?"
    "Why is it that I say I have a session, and I don't?"

His response: `If your connection is laggy, or for whatever reason it takes a while for it to upload the next bit, the components haven't been loaded. So if you wait a while, it will probably work.`.

**Why not add in a verbose messages saying: `Loading stdapi` and/or `Loaded stdapi`?**
The could be in a different style (colour/size/font/status tag) to signal that it's a background action (kind of like with Metasploit console & verbose.
I for one, would love to know what's happening behind the scenes more. I don't mind the extra lines/spam - I would rather know what's going on/what's happening/when it's happening.

If you keep watching/watch from the start - it would be great to have an in-built option, showing what's happening in a little more depth in general (e.g. with the migrate command too).

- - -

After talking to @OJ about it, two points came up: Make sure its for the active/foreground session & feedback to users who type too early.

```
<TheColonial> you really only want that for a foreground handler
<TheColonial> background ones you just want a single notification
```

```
<TheColonial> that if you type a command too early..
<TheColonial> the session says "Patience, Padawan, the session is still wiring up"
<TheColonial> ie. it gives a status
<TheColonial> (still uploading stdapi... etc)
<TheColonial> some random banners would be fun
<TheColonial> "Are you always this impatient?"
<TheColonial> "You tried accessing your session before it was ready. This incident will be reported"
```
